staging/src/k8s.io/apiserver/pkg/endpoints/request/webhook_duration.go

```diff
@@ -64,6 +75,14 @@ func (t *durationTracker) Track(f func()) {
 	f()
 }
 
+// TrackWithDuration tracks latency from the given duration
+// using aggregateFunction
+func (t *durationTracker) TrackWithDuration(d time.Duration) {
```

Comment:
nit: I prefer `TrackDuration` without the "with".

Reply:
done